packages/@scope/jsii-calc-base-of-base/tsconfig.json

```diff
@@ -0,0 +1,58 @@
+{
+  "compilerOptions": {
+    "alwaysStrict": true,
+    "charset": "utf8",
+    "declaration": true,
+    "experimentalDecorators": true,
+    "inlineSourceMap": true,
+    "lib": ["es2018"],
+    "noEmitOnError": true,
+    "noFallthroughCasesInSwitch": true,
+    "noImplicitAny": true,
+    "noImplicitReturns": true,
+    "noImplicitThis": true,
+    "noUnusedLocals": true,
+    "noUnusedParameters": true,
+    "resolveJsonModule": true,
+    "strict": true,
+    "strictNullChecks": true,
+    "strictPropertyInitialization": true,
+    "stripInternal": false,
+    "composite": true,
+    "incremental": true
+  },
+  "include": ["**/*.ts"],
+  "exclude": ["node_modules"],
+  // The JSII configuration
+  "x-aws-jsii": {
```

Comment:
I wouldn't use the "aws" name here. Let's call this "x-jsii" or just "jsii" (no need to be too pure about this).

Reply:
Yeah I guess `x-jsii` is good... (The `x-` prefix signifies "extension")